e2e/tests/smoke/block_storage_smoke_test.go

```diff
@@ -40,9 +40,9 @@ func (suite *BlockStorageTestSuite) SetupTest() {
 
 	require.Nil(suite.T(), err)
 
-	rookInfra.ValidateAndSetupTestPlatform()
+	rookInfra.ValidateAndSetupTestPlatform(env.InstallRook == "true")
```

Comment:
env.InstallRook is a flag that comes from CLI parameters, that is why its a String. Extracting the logic out of function call to make code cleaner.